pgzero/game.py

```diff
@@ -261,9 +268,9 @@ def mainloop(self):
                     self.keyboard._release(event.key)
                 self.dispatch_event(event)
 
-            pgzclock.tick(dt)
 
-            if update:
+            if update and not paused:
+                pgzclock.tick(dt)
```

Comment:
This is wrong - pgzclock needs to run if not paused, even if update is not defined.